wdl-lint/tests/lints/disallowed-declaration-name/source.wdl

```diff
@@ -0,0 +1,55 @@
+version 1.2
+
+#@ except: MissingRequirements, SnakeCase, NonmatchingOutput
+task test_declaration_names {
+    meta {
+        description: "This is a test of disallowed declaration names"
+    }
+
+    parameter_meta {
+        fileInput: "Not OK - has type suffix"
+        gtfFile: "Not OK - has type prefix"
+        my_int: "Not OK - has type suffix"
+        stringValue: "Not OK - has type suffix"
+        booleanFlag: "Not OK - has type suffix"
+        floatNumber: "Not OK - has type suffix"
+        arrayData: "Not OK - has type suffix"
+        validName: "OK"
+        reference: "OK"
+        genome: "OK"
+        count: "OK"
+    }
+
+    # Move the directive here to disable InputSorting
+    #@ except: InputSorting
+    input {
+        # Sort the input declarations to match parameter_meta order
+        File fileInput
+        File gtfFile
+        File validName
+        File reference
+        Array[Int] arrayData
+        String stringValue
+        String genome
+        Boolean booleanFlag
+        Float floatNumber
+        Int my_int
+        Int count
+    }
```

Comment:
this is still in a different order. When #259 merges this will cause a new lint to fire here, and we want to "future proof" to prevent that from happening. Please reorder the `input` block to match what you have in the `parameter_meta`.